tls/s2n_record_read.c

```diff
@@ -98,16 +98,144 @@ int s2n_record_header_parse(struct s2n_connection *conn, uint8_t * content_type,
     return 0;
 }
 
+int s2n_record_read_decryption_non_aead(struct s2n_connection *conn, const struct s2n_cipher_suite *cipher_suite, struct s2n_session_key *session_key,
```

Comment:
What do you think about changing this to "decrypt_non_aead" to make it more of a verb describing what the function is doing?